WARNING  viperlog:logger.py:109 Unsupported units found for column/variable TIME: ['dubious/units']. Cannot create an astropy.units.Units object from it: 'dubious/units' did not parse as unit: At col 0, dubious is not a valid unit.  If this is meant to be a custom unit, define it with 'u.def_unit'. To have it recognized inside a file reader or other code, enable it with 'u.add_enabled_units'. For details, see https://docs.astropy.org/en/latest/units/combining_and_defining.html
WARNING  viperlog:logger.py:109 Invalid units found for column/variable DATA: (None, None)
WARNING  viperlog:logger.py:109 Unsupported units found for column/variable DATA: (None, None). Cannot create an astropy.units.Units object from it: None cannot be converted to a Unit
WARNING  viperlog:logger.py:109 Invalid units found for column/variable TIME_CENTROID: (3.1,)